dubbo-common/src/main/java/org/apache/dubbo/config/annotation/DubboService.java

```diff
@@ -323,4 +323,9 @@
      * @return
      */
     String executor() default "";
+
+    /**
+     * Payload max length.
+     */
+    String payload() default "";
```

Comment:
Why not use the int type, consistent with the previous one?

Reply:
> Why not use the int type, consistent with the previous one?

Thank you. Yes, I started using int and later made changes to be consistent with dubbo.xsd